base_geoengine/geo_model.py

```diff
@@ -41,8 +41,10 @@ def fields_get(self, allfields=None, attributes=None):
 
     @api.model
     def _get_geo_view(self):
-        geo_view = self.env["ir.ui.view"].search(
-            [("model", "=", self._name), ("type", "=", "geoengine")], limit=1
+        geo_view = (
```

Comment:
Why not:
```
geo_view = self.sudo().env["ir.ui.view"].search(
    ("model", "=", self._name), ("type", "=", "geoengine")],
    limit=1
)
```

Reply:
Two line less code and no unneeded brackets around the assignment expression.

Reply:
Because Black won't let me :cry:

Reply:
@aleuffre I see, this works though:
```
        view_model = self.env["ir.ui.view"]
        geo_view = view_model.sudo().search(
            [("model", "=", self._name), ("type", "=", "geoengine")],
            limit=1,
        )
```